packages/core/src/test/awsService/cloudWatchLogs/registry/logDataRegistry.test.ts

```diff
@@ -141,7 +140,7 @@ describe('LogDataRegistry', async function () {
             return async function (
                 logGroupInfo: CloudWatchLogsGroupInfo,
                 parameters: CloudWatchLogsParameters,
-                nextToken?: CloudWatchLogs.NextToken
+                nextToken?: string
```

Comment:
I guess the v3 loses these types?

Reply:
Yeah, they were simple aliases in v2 so not losing anything meaningful. https://github.com/aws/aws-sdk-js/blob/966fa6c316dbb11ca9277564ff7120e6b16467f4/clients/cloudwatchlogs.d.ts#L2330